### PR TITLE
Create sv.js

### PR DESCRIPTION
**Pull request by prince17dec**
Swedish Language support.

### CODE REVIEW

File: src/locale/sv.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+  accepted             : 'Fält :attribut måste accepteras.',
```
Comment (stuyam) [inline src/locale/sv.js:1]: You are missing the opening and closing lines of the object, check this file as an example: https://github.com/dockwa/simple-react-validator/blob/master/src/locale/el.js#L1-L2

File: src/locale/sv.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+ SimpleReactValidator.addLocale('el', {
```
Comment (stuyam) [inline src/locale/sv.js:1]:
```suggestion
// Swedish
SimpleReactValidator.addLocale('el', {
```

File: src/locale/sv.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+ // Swedish
+ SimpleReactValidator.addLocale('el', {
```
Comment (stuyam) [inline src/locale/sv.js:2]:
```suggestion
 SimpleReactValidator.addLocale('sv', {
```

**Review by prince17dec — COMMENTED** [review]
Resolve comments

**Pull request merged**